src/data/latex/agda.sty

```diff
@@ -2,7 +2,11 @@
 % Some useful commands when doing highlighting of Agda code in LaTeX.
 % ----------------------------------------------------------------------
 
+% !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+% !!! NOTE: when you make changes to this file, bump the date. !!!
+% !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 \ProvidesPackage{agda}
+  [2021/07/14 version 2.6.3 Formatting LaTeX generated by Agda]
```

Comment:
Can we use the [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601#Dates) instead? If "14" had been "12" the date above would have been ambiguous.

Reply:
We should go with the convention.
From: https://en.wikibooks.org/wiki/LaTeX/Creating_Packages#Creating_your_own_package
> The <version> should begin with a date in the format YYYY/MM/DD.